packages/amplify-graphql-docs-generator/package.json

```diff
@@ -38,13 +38,16 @@
     "typescript": "^3.5.1"
   },
   "dependencies": {
+    "aws-sdk": "^2.475.0",
```

Comment:
Are these graphql transformer package changes your changes?

Reply:
no - they should be reverted now.